core/exception/no_memory_error_spec.rb

```diff
@@ -0,0 +1,5 @@
+require_relative '../../spec_helper'
+
+describe "NoMemoryError" do
+  # Nothing to do here.
+end
```

Comment:
I'm thinking that every built-in exception should be represented here, even if there's nothing to do.

Reply:
If there is no test/spec I don't see the point, please remove them.
Those are already listed in hierarchy_spec.rb which implicitly tests they exist.